### PR TITLE
chore: describe `ast_helpers` exposed in Python challenges

### PR DESCRIPTION
**Pull request by gikf**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a [descriptive title](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request?id=prepare-a-good-pr-title) (**not** a vague title like `Update index.md`)

- - -
- Not sure if it's clear `ast_helpers` module in challenges is not an alternate way to write test, but convenience for writing/debugging the ast part of the test.

### CODE REVIEW

File: src/content/docs/curriculum-help.mdx [hunk 1]
```diff
@@ -197,6 +197,25 @@ To check the return value of `__str__` you would write:
 });
 ```
 
+AST-based helpers are also exposed in the Python challenges in `ast_helpers` module. `_Node` class corresponds to the `ast_helpers.Node`:
```
Comment (majestic-owl448) [inline src/content/docs/curriculum-help.mdx:200]: you still need to run the python code through pyodide, right?

Comment (gikf) [reply src/content/docs/curriculum-help.mdx:200]: Uhm, not by yourself. What I'm trying to communicate is this:

1. Go to python challenge, ie.: https://www.freecodecamp.org/learn/scientific-computing-with-python/learn-how-to-work-with-numbers-and-strings-by-implementing-the-luhn-algorithm/step-15
2. import `ast_helpers`
3. Use `ast_helpers.Node` to parse code and play around with various methods of the class.

Comment (majestic-owl448) [reply src/content/docs/curriculum-help.mdx:200]: you mean to use the ast_helpers in the editor?

Comment (gikf) [reply src/content/docs/curriculum-help.mdx:200]: Yeah

**Review by majestic-owl448 — APPROVED** [review]

**Pull request merged**